src/policies.rs

```diff
@@ -122,21 +123,25 @@ fn verify_different_authors<G: Git>(commits: &Vec<Commit<'_>>) -> Result<(), Box
 }
 
 fn verify_email_addresses(author_domain: &str,committer_domain: &str, commits: &Vec<Commit<'_>>) -> Result<(), Box<dyn Error>> {
```

Comment:
When we check whether the `commit.author` is in the `author_domain` and it is not in there then we will never try to check the `match commit.committer()` codeblock if I am not mistaken. 

Is this how we want to do it?

Reply:
This is as designed